scripts/src/aica-docker-completion.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+SCRIPTS=("connect" "interactive" "server")
+CONTAINERS=$(docker ps --format "{{ .Names }}")
+IMAGES=$(docker images --format "{{.Repository}}:{{.Tag}}")
+IMAGES=${IMAGES//"<none>:<none>"/}
+
+_aica_docker () {
+  local cur prev
+  COMPREPLY=()
+  cur="${COMP_WORDS[COMP_CWORD]}"
+  prev="${COMP_WORDS[COMP_CWORD-1]}"
+
+  if [[ ${COMP_CWORD} -eq 2 ]]; then
```

Comment:
I haven't looked into the syntax of autocompletion too much yet, but if you regenerate the variables here on the completion call, is it needed to populate them in advance at the top of the file? If it is needed in both places, you could reduce code duplication by setting these variables in a sub function, though that's a bit awkward to parse and would increase the verbosity too. Mainly I want to check that you don't define it twice if you don't need to.

Reply:
of course, thanks for the hint